src/erlcloud_ses.erl

```diff
@@ -132,6 +145,147 @@ delete_identity(Identity, Config) ->
         {error, Reason} -> {error, Reason}
     end.
 
+%%------------------------------------------------------------------------------
+%% @doc
+%% Custom Verification Templates
+%% SES API:
+%% [https://docs.aws.amazon.com/ses/latest/APIReference/API_GetCustomVerificationEmailTemplate.html]
+%%
+%% Template attributes:
+%%   { template_name , string() }
+%%   { from_email_address , string() }
+%%   { template_subject , string() }
+%%   { template_content , string() }     -- please see notes in API Guide on what is allowed
+%%   { success_redirect_url , string() }
+%%   { failure_redirect_url , string() }
+%%
+%% On template creation, all attributes are mandatory.
+%% On template updates, only include the attributes you need to modify
+%% When listing templates, all attributes are included except template_content, use the get function to retrieve it
+%%
+%%  create_custom_verification_email_template
+%%  update_custom_verification_email_template
+%%  send_custom_verification_email
+%%  delete_custom_verification_email_template
+%%  get_custom_verification_email_template
+%%  list_custom_verification_email_templates
+%%
+%% @end
+%%------------------------------------------------------------------------------
+
+-type custom_template_attribute_names() :: template_name | from_email_address | template_subject | template_content | success_redirect_url | failure_redirect_url .
+-type custom_template_attributes() :: [ { custom_template_attribute_names() , string() } ].
+
+-type create_custom_verification_email_template_result() :: ok | { error , term() }.
+
+-spec create_custom_verification_email_template( string() , string() ,string() ,string() ,string() ,string() ) -> create_custom_verification_email_template_result().
+create_custom_verification_email_template( TemplateName , FromEmailAddress , TemplateSubject , TemplateContent , SuccessRedirectionURL , FailureRedirectionURL ) ->
+  create_custom_verification_email_template( TemplateName , FromEmailAddress , TemplateSubject , TemplateContent , SuccessRedirectionURL , FailureRedirectionURL , default_config() ).
+
+-spec create_custom_verification_email_template( string() , string() ,string() ,string() ,string() ,string() , aws_config() ) -> create_custom_verification_email_template_result().
+create_custom_verification_email_template( TemplateName , FromEmailAddress , TemplateSubject , TemplateContent , SuccessRedirectionURL , FailureRedirectionURL ,Config ) ->
+  Params = encode_params(
+            [   { template_name , TemplateName} ,
+                { from_email_address , FromEmailAddress } ,
+                { template_subject , TemplateSubject } ,
+                { template_content , TemplateContent } ,
+                { success_redirect_url , SuccessRedirectionURL} ,
+                { failure_redirect_url , FailureRedirectionURL} ] ),
+  case ses_request(Config, "CreateCustomVerificationEmailTemplate", Params) of
+    {ok, _Doc} ->
+      ok;
```

Comment:
stylish: either both on new line or both on the same line

Reply:
OK. So I stuck now with what I have seen. Short stays on the same line, long, 2+ lines. Next checkin.